firebase-android-release-dashboard/src/components/Release/ReleaseDetails/ReleaseDetails.js

```diff
@@ -0,0 +1,94 @@
+import {Grid, Link, Typography} from "@material-ui/core";
+import {Alert} from "@material-ui/lab";
+import {GitBranchIcon} from "@primer/octicons-react";
+import PropTypes from "prop-types";
+import React from "react";
+import {RELEASE_STATES} from "../../../utils/releaseStates.js";
+import BuildArtifacts from "../BuildArtifacts/BuildArtifacts.js";
+import GithubChecks from "../GithubChecks/index.js";
+import ReleaseLibraries from "../ReleaseLibraries/ReleaseLibraries.js";
+import useStyles from "./styles.js";
+
+// Sort libraries alphabetically by name
+const sortLibraries = (libraries) => {
+  return [...libraries]
+      .sort((a, b) => a.libraryName.localeCompare(b.libraryName));
+};
+
+/**
+ * Component to display details of a release that has release data.
+ *
+ * All releases that are not in the "scheduled" state have release data.
+ *
+ * @param {Object} release - An object containing release details.
+ * @return {JSX.Element} The rendered JSX element.
+ */
+function ReleaseDetails({release}) {
+  const classes = useStyles();
+
+  const sortedLibraries = sortLibraries(release.libraries);
+
+  if (release.state === RELEASE_STATES.ERROR) {
+    return (
+      <Alert severity="error">
+        This release is in an error state. Please contact the release operator.
+      </Alert>
```

Comment:
Should this be an if statement? if we are logged in, shouldn't this return the actual error?

Reply:
We aren't storing the cause of the errors in the backend yet, we only know that the release is in an error state.

Would you like me to create a PR to add an `errorMessage` field in Firestore releases, and add this feature to the PR, or should I merge this and work on that later?

Reply:
You can merge and do it later